packages/preferences/src/browser/preferences-contribution.ts

```diff
@@ -114,6 +115,11 @@ export class PreferencesContribution extends AbstractViewContribution<Preference
                 widget.setScope(PreferenceScope.Workspace);
             }
         });
+        commands.registerCommand(PreferencesCommands.OPEN_FOLDER_PREFERENCES, {
+            isEnabled: () => !!this.workspaceService.isMultiRootWorkspaceOpened && this.workspaceService.tryGetRoots().length > 0,
+            isVisible: () => !!this.workspaceService.isMultiRootWorkspaceOpened && this.workspaceService.tryGetRoots().length > 0,
+            execute: () => this.openFolderPreferences(root => { this.openView({ activate: true }); this.scopeTracker.setScope(root.resource); })
```

Comment:
minor - for clarify and consistency:

```suggestion
            execute: () => { 
              this.openFolderPreferences(root => { this.openView({ activate: true }); 
              this.scopeTracker.setScope(root.resource); 
            })
```